index.html

```diff
@@ -4,10 +4,49 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Personal Portfolio</title>
+    <title>Document</title>
     <link rel="stylesheet" href="style.css">
 </head>
-<body>
-    
+<body class="flex_column">
+    <header>
+        <nav class="flex">
+            <div class="logo flex_column">
+                <a href=""><img src="assets/celec.png" alt="logo"></a>
+                <p>My logo</h2>
+            </div>
+            <a href="#" class="burgeur flex"><img src="assets/Icon - Menu.png" alt="menu"></a>
```

Comment:
- [ ] As the previous reviewer suggested, the hamburger menu should be wrapped in its own `nav` tag. since it contains the list of navigation links, it, therefore, works as a nav button. For this case, you may consider removing the logo from the `nav` tag. Additionally, I have noticed that there are no image files in your project. Kindly consider committing them to your remote repository ☺️.
- [ ] Optionally, consider using the [relative image path](https://www.w3schools.com/html/html_filepaths.asp). For example `<img src="../image_store/Part2.jpg" alt="Dress Picture"/>`